.github/dependabot.yml

```diff
@@ -15,7 +15,7 @@ updates:
       changesets:
         patterns: ['@changesets/*']
       eslint:
-        patterns: ['eslint', 'eslint-*']
+        patterns: ['eslint-*'] # TODO: Include 'eslint' when we're ready for ESLint v9.
```

Comment:
Can we only ignore major updates instead?

ref https://github.blog/changelog/2021-05-21-dependabot-version-updates-can-now-ignore-major-minor-patch-releases/

Reply:
Ah, much better 👍🏼 Please open a PR 🙏🏼